quna/build.gradle

```
@@ -19,6 +19,7 @@ dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-aop'
     runtimeOnly 'com.h2database:h2'
     testImplementation 'org.springframework.boot:spring-boot-starter-test'
+    implementation 'io.jsonwebtoken:jjwt:0.9.1'
```

`:0.9.1`은 없어도 될 것 같아요~~
```suggestion
    implementation 'io.jsonwebtoken:jjwt'
```